collectors/nvd/collectors.py

```diff
@@ -232,41 +233,43 @@ def collect(self, cve: Union[str, None] = None) -> str:
         for item in batch_data:
             cve_id = item["cve_id"]
 
-            if self.snippet_creation_enabled:
-                try:
-                    Snippet.objects.get(
-                        source=Snippet.Source.NVD, content__cve_id=cve_id
-                    )
-                except Snippet.DoesNotExist:
-                    if should_create_snippet(item["description"]):
-                        snippet = Snippet(source=Snippet.Source.NVD, content=item)
-                        snippet.save()
-                        new_snippets.append(cve_id)
-
+            # check if a flaw is already in DB, and if so, update CVSS if necessary
             try:
                 flaw = Flaw.objects.get(cve_id=cve_id)
                 # update NVD CVSS2 or CVSS3 data if necessary
                 updated_in_flaw = self.update_cvss_via_flaw(flaw, item)
                 updated_in_flawcvss = self.update_cvss_via_flawcvss(flaw, item)
 
                 if updated_in_flaw or updated_in_flawcvss:
-                    desync.append(cve_id)
+                    updated_flaws.append(cve_id)
                     # no automatic timestamps as those go from Bugzilla
                     # and no validation exceptions not to fail here
                     flaw.save(auto_timestamps=False, raise_validation_error=False)
             except Flaw.DoesNotExist:
                 pass
 
-        logger.info(
-            f"NVD CVSS scores were updated for the following CVEs: {', '.join(desync)}"
-            if desync
-            else "No CVEs with desynced NVD CVSS."
-        )
-        logger.info(
-            f"New snippets were created for the following CVEs: {', '.join(new_snippets)}"
-            if new_snippets
-            else "No new snippets."
+            # create a new snippet with a flaw (if it does not exist) and link them
+            snippet = None
+            if self.snippet_creation_enabled:
```

Comment:
I discussed with @Elkasitu that `snippet_creation_enabled` should be rewritten into env variable.